src/components/Combos/Combos.jsx

```diff
@@ -77,7 +77,12 @@ const HeroSelector = ({
   heroName
 }) => (
   <StyledHeroSelector selected={selected} isFiltered={isFiltered}>
-    <HeroImage id={id} imageSizeSuffix={IMAGESIZE_ENUM.VERT.suffix}/>
+    <HeroImage
+      id={id} 
+      imageSizeSuffix={IMAGESIZE_ENUM.SMALL.suffix} 
+      style={{position: 'absolute', width: 200, left: -30, opacity: .5, filter: 'blur(5px)'}}
+    />
+    <HeroImage id={id} imageSizeSuffix={IMAGESIZE_ENUM.SMALL.suffix} style={{width: 'auto', height: 40, zIndex: 2}}/>
```

Comment:
two heroimages?

Reply:
Yes, one image is filler/padding because I didn't want to zoom in too much and keep the aspect ratio.

Reply:
![Uploading cc2.png…]()